src/patternfly/components/Page/page.scss

```diff
@@ -272,50 +279,51 @@
   margin-bottom: var(--pf-c-page__header-tools--MarginBottom);
   margin-left: auto; // to push it to the right
 
-  .pf-m-icons {
+  // Hides on mobile visible on desktop
+  .pf-m-desktop {
     display: none;
     visibility: hidden;
 
     @media screen and (min-width: $pf-global--breakpoint--lg) {
-      display: block;
+      display: inline-block;
       visibility: visible;
     }
   }
 
+  // Hides on desktop visible on mobile
   .pf-m-mobile {
-    margin-right: 0;
+    display: inline-block;
+    visibility: visible;
 
     @media screen and (min-width: $pf-global--breakpoint--lg) {
       display: none;
       visibility: hidden;
     }
   }
 
-  .pf-m-user {
```

Comment:
removing this is also a breaking change.

Reply:
is removing pf-m-user a breaking change now that the display is handled by its parent?

Reply:
Yep, think about people who aren't using react and pull in these changes without updating their markup. A good way to test this is to copy the stylesheet from this PR into a local build of master and test all of the page component pages in the dev workspace. Everything should work as it did before this update.